Presentation.Web/app/components/data-processing/tabs/data-processing-registration-tab-main.controller.ts

```diff
@@ -97,10 +93,20 @@
 
         private bindIsAgreementConcluded() {
             this.isAgreementConcluded = {
-                selectedElement: this.dataProcessingRegistration.agreementConcluded.value,
-                select2Options: new Models.ViewModel.Shared.YesNoIrrelevantOptions().options,
+                selectedElement: this.getYesNoIrrelevantOptionFromId(this.dataProcessingRegistration.agreementConcluded.value),
+                select2Config: this.select2LoadingService.select2LocalDataNoSearch(() => new Models.ViewModel.Shared.YesNoIrrelevantOptions().options, false),
                 elementSelected: (newElement) => this.changeIsAgreementConcluded(newElement)
             };
+            this.shouldShowAgreementConcludedAt =
+                this.isAgreementConcluded.selectedElement &&
+                this.isAgreementConcluded.selectedElement.optionalObjectContext === Models.Api.Shared.YesNoIrrelevantOption.YES;
+        }
+
+        private getYesNoIrrelevantOptionFromId(id?: number): Models.ViewModel.Generic.Select2OptionViewModel<Models.Api.Shared.YesNoIrrelevantOption> {
+            if (id === null) {
```

Comment:
@JacobMalling JEg har rettet det. Det er forkert at sende den "falske" option tilbage. Jeg prøver desuden lige at undersøge hvorfor vi ikke har adgang til nyere TS features. Vi kører 3.9 men har ikke adgang til null propagation (optional chaining) som burde være tilgændeligt fra 3.7. Det gør at vi kan hej?.då?.der() i stedet for hej && hej.då && hej.då.der()

Reply:
@JacobMalling Ja så gav det mening. I VS har den fat i en TS compiler v 3.9 men vores npm pakke peger på 
![image](https://user-images.githubusercontent.com/13116889/94230202-c007ac80-ff01-11ea-8d6d-6fcd912ec75a.png)